site/profile/manifests/core/anaconda.pp

```diff
@@ -0,0 +1,78 @@
+# @summary
+#   Installs anaconda
+#
+# @param python_env_name
+#   if exist, sets the name of the virtual environment
+#
+# @param python_env_version
+#   if exist, sets the version of python for the virtual environment
+#
+# @param conda_packages
+#   if contains at least 1 tuple (name and channel), will install the package
+#    
+class profile::core::anaconda (
+  String $python_env_name,
```

Comment:
It is usually preferable to use `String[1]` as `String` will allow an empty string. E.g. `""`